00-platform/01-jupyter.ipynb

```diff
@@ -46,6 +46,7 @@
     "\n",
     "``` bash\n",
     "$ cd ~/2018-03-06-ibioic\n",
+    "$ source activate 2018-03-06-ibioic\n",
```

Comment:
I thought we'd got rid of the environment?

Reply:
It was still being made reference to in the paragraph below:

> The second command source activates something called a Python Virtual Environment, which takes care of all the Python library dependencies we'll be using - including Jupyter.

The choice was to remove that line or to put the `source` command back in. I'd forgotten we removed source when I chose to put the command back in.

Reply:
Re: generating HTML

`File -> Download As -> HTML`

This will put the file, unhelpfully, in your `Download` directory by default. You'll have to copy it from there to the repo.